content/en/flux/gitops-toolkit/packages.md

```diff
@@ -187,7 +188,7 @@ func main() {
   ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
   defer cancel()
 
-  // create a Helm repository pointing to Bitnami
+  // create a Helm repository pointing to https://kubernetes.github.io/ingress-nginx
```

Comment:
why the change here? The repository points to bitnami still.

Reply:
Good catch, i wanted to change the repo but installing ingress-nginx in the default ns doesn't make sense. We should rewrite this with OCIRepo, but that's for another time.